frontend/src/App.tsx

```diff
@@ -7,27 +7,41 @@ import { MissionFilterProvider } from 'components/Contexts/MissionFilterContext'
 import { MissionsProvider } from 'components/Contexts/MissionListsContext'
 import { SafeZoneProvider } from 'components/Contexts/SafeZoneContext'
 import { AlertProvider } from 'components/Contexts/AlertContext'
+import { SignalRProvider } from 'components/Contexts/SignalRContext'
+import { RobotProvider } from 'components/Contexts/RobotContext'
+import { AuthProvider } from 'components/Contexts/AuthProvider'
+import { InstallationProvider } from 'components/Contexts/InstallationContext'
 
 const App = () => (
     <AlertProvider>
-        <SafeZoneProvider>
-            <MissionsProvider>
-                <LanguageProvider>
-                    <MissionControlProvider>
-                        <UnauthenticatedTemplate>
-                            <div className="sign-in-page">
-                                <AssetSelectionPage></AssetSelectionPage>
-                            </div>
-                        </UnauthenticatedTemplate>
-                        <AuthenticatedTemplate>
-                            <MissionFilterProvider>
-                                <FlotillaSite />
-                            </MissionFilterProvider>
-                        </AuthenticatedTemplate>
-                    </MissionControlProvider>
-                </LanguageProvider>
-            </MissionsProvider>
-        </SafeZoneProvider>
+        <MissionsProvider>
+            <LanguageProvider>
+                <MissionControlProvider>
+                    <UnauthenticatedTemplate>
+                        <div className="sign-in-page">
+                            <AssetSelectionPage></AssetSelectionPage>
+                        </div>
+                    </UnauthenticatedTemplate>
+                    <AuthenticatedTemplate>
+                        <MissionFilterProvider>
+                            <InstallationProvider>
+                                <AuthProvider>
+                                    <SignalRProvider>
+                                        <RobotProvider>
+                                            <MissionsProvider>
```

Comment:
Why is MissionProvider two places?

Reply:
I just took things out of flotillaSite so I guess they were duplicated